packages/block-library/src/navigation-link/edit.js

```diff
@@ -80,57 +45,23 @@ function NavigationLinkEdit( {
 	insertLinkBlock,
 } ) {
 	const { label, opensInNewTab, title, url, nofollow, description } = attributes;
-	const link = title ? { title: unescape( title ), url } : null;
-	const [ isLinkOpen, setIsLinkOpen ] = useState( ! label && isSelected );
-
-	let onCloseTimerId = null;
+	const link = {
+		title: title ? unescape( title ) : '',
+		url,
+		newTab: opensInNewTab,
```

Comment:
Would it make sense to name the property opensInNewTab like the attribute?

Reply:
We can I just used the default value precedently used.